src/Orleans/Properties/AssemblyInfo.cs

```diff
@@ -54,4 +54,5 @@ MIT License
 [assembly: InternalsVisibleTo("UnitTestGrainInterfaces, PublicKey=0024000004800000940000000602000000240000525341310004000001000100eb81539dc2728946d4860bad0fbc91ea23d7a835e00547aa4e9a7abf185535b564d29dbe77915b779951fef2f4b1ab39cc3a246cf572c7b217b5c892835a687eb22638183d66930039ff9fa6e7e71cb880f854fb76b884dfc905bf1f3c623abad7d4732c1ce62efff937155e352170831239acbe103a41ed363a90001d3108ca")]
 [assembly: InternalsVisibleTo("UnitTestGrains, PublicKey=0024000004800000940000000602000000240000525341310004000001000100eb81539dc2728946d4860bad0fbc91ea23d7a835e00547aa4e9a7abf185535b564d29dbe77915b779951fef2f4b1ab39cc3a246cf572c7b217b5c892835a687eb22638183d66930039ff9fa6e7e71cb880f854fb76b884dfc905bf1f3c623abad7d4732c1ce62efff937155e352170831239acbe103a41ed363a90001d3108ca")]
 [assembly: InternalsVisibleTo("TesterInternal, PublicKey=0024000004800000940000000602000000240000525341310004000001000100eb81539dc2728946d4860bad0fbc91ea23d7a835e00547aa4e9a7abf185535b564d29dbe77915b779951fef2f4b1ab39cc3a246cf572c7b217b5c892835a687eb22638183d66930039ff9fa6e7e71cb880f854fb76b884dfc905bf1f3c623abad7d4732c1ce62efff937155e352170831239acbe103a41ed363a90001d3108ca")]
+[assembly: InternalsVisibleTo("Tester, PublicKey=0024000004800000940000000602000000240000525341310004000001000100eb81539dc2728946d4860bad0fbc91ea23d7a835e00547aa4e9a7abf185535b564d29dbe77915b779951fef2f4b1ab39cc3a246cf572c7b217b5c892835a687eb22638183d66930039ff9fa6e7e71cb880f854fb76b884dfc905bf1f3c623abad7d4732c1ce62efff937155e352170831239acbe103a41ed363a90001d3108ca")]
```

Comment:
Nope! Tester project should not have internal visibility into Orleans. That is what TesterInternal is for. The code in Tester projects should only have public access to Orleans dlls.